### PR TITLE
Add possibility configure path cookie

### PR DESCRIPTION
**Pull request by anijra**

### CODE REVIEW

**Comment by clash82** [comment]
@anijra could you please revert files permission change?

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -25,6 +25,9 @@ public function getConfigTreeBuilder()
                     ->defaultValue('365')
                     ->info('how many days banner should be hidden when user accepts policy?')
                 ->end()
+                ->integerNode('path')
```
Comment (clash82) [inline DependencyInjection/Configuration.php]: AFAIK this is a `scalarNode`

would be great if you could also provide some details using `info` key

File: README.md [hunk 2]
```diff
@@ -92,6 +92,7 @@ Parameter        | Default value                                  | Description
 ---------------- | ---------------------------------------------- | -----------
 cookieName       | privacyCookieAccepted                          | Sets your own status cookie name
 cookieValidity   | 365                                            | Says how many days privacy banner should be hidden when user accepts policy?
+cookiePath       | null                                           | Specifies the cookie path
```
Comment (clash82) [inline README.md]: We could add some context here:

` Specifies the cookie path (by default cookie will be available only for the current domain)`

File: Resources/config/default_settings.yml [hunk 3]
```diff
@@ -3,6 +3,8 @@ parameters:
     ez_privacy_cookie.days: 365
     # name to be used to store cookie status
     ez_privacy_cookie.cookie_name: privacyCookieAccepted
+    # Specifies the cookie path
```
Comment (clash82) [inline Resources/config/default_settings.yml]: please, use lowercase to be consistent with rest of the comments here ;)

File: Resources/public/js/privacycookie.js [hunk 4]
```diff
@@ -60,19 +61,25 @@
...
      */
-    eZ.PrivacyCookieBanner.prototype.setCookie = function (name, value, days) {
+    eZ.PrivacyCookieBanner.prototype.setCookie = function (name, value, days, path) {
         var d = new Date();
         d.setTime(d.getTime() + (days * 24 * 60 * 60 * 1000));
         var expires = 'expires=' + d.toUTCString();
-        document.cookie = name + '=' + value + '; ' + expires;
+
+        var cookiePath = '';
+        if(path) {
```
Comment (clash82) [inline Resources/public/js/privacycookie.js]: CS: `if (path) {`

ref. https://github.com/airbnb/javascript#whitespace--around-keywords

**Review by clash82 — CHANGES_REQUESTED** [review]
Good work, some small improvements submitted

**Review by clash82 — APPROVED** [review]

**Review by damianz5 — APPROVED** [review]

**Comment by andrerom** [comment]
Still some chmod changes here!

**Comment by clash82** [comment]
@anijra two files are still waiting for permissions change

**Review by SylvainGuittard — APPROVED** [review]
Thank you @anijra for the contribution

**Comment by SylvainGuittard** [comment]
@anijra could you please rebase your PR in order to have just one commit. I will merge it after.

**Comment by clash82** [comment]
@SylvainGuittard please, don't forget to create a net version tag.

**Pull request merged**